README.md

```diff
@@ -19,3 +20,7 @@ to be the most efficient one!
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+## My Response
+
+The $\Theta$ should be $\Theta(V^2)$ because in the worst case it would need to go through each node twice, until it finds the shortest distance.
```

Comment:
Not following you here. Each node could be processed more than twice.

Reply:
Ah ok, so am I going to have to take into consideration the searching as well? Giving me something like (V + E) + (V^2)?  because it would need to go through ever node and edge, and repeat this until the shortest distance is found?

Reply:
That would still result in the same overall complexity though, wouldn't it?